bento/src/main/java/com/yelp/android/bento/core/AsyncInflationBridge.kt

```diff
@@ -0,0 +1,163 @@
+package com.yelp.android.bento.core
+
+import android.view.View
+import androidx.annotation.VisibleForTesting
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.OnLifecycleEvent
+import androidx.lifecycle.lifecycleScope
+import androidx.recyclerview.widget.RecyclerView
+import com.yelp.android.bento.componentcontrollers.RecyclerViewComponentController.constructViewHolder
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.ConcurrentLinkedDeque
+import java.util.concurrent.Executors
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.asCoroutineDispatcher
+import kotlinx.coroutines.async
+import kotlinx.coroutines.awaitAll
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+import kotlinx.coroutines.withContext
+
+/**
+ * This acts as a bridge between RecyclerViewComponentController and the underlying
+ * RecyclerView.Adapter. It intercepts the addComponent() calls and inflates views with
+ * [BentoAsyncLayoutInflater]. Components are added to the RecyclerView when the inflation
+ * is finished.
+ */
+internal class AsyncInflationBridge(
```

Comment:
If you write it like `internal class AsyncInflationBridge @JvmOverloads constructor(`, then in the java code, you would be able to simply call `AsyncInflationBridge(recyclerView)`

Reply:
Thanks!